src/components/SlateEditor/plugins/table/utils.ts

```diff
@@ -0,0 +1,345 @@
+import React from 'react';
+import { Editor, Element, NodeEntry, Path, Transforms } from 'slate';
+import { jsx } from 'slate-hyperscript';
+import { ReactEditor } from 'slate-react';
+import {
+  TableCellElement,
+  TableElement,
+  TableRowElement,
+  KEY_ARROW_DOWN,
+  KEY_ARROW_UP,
+  KEY_TAB,
+} from '.';
+import { defaultParagraphBlock } from '../paragraph/utils';
+
+export const TYPE_TABLE = 'table';
+export const TYPE_TABLE_ROW = 'table-row';
+export const TYPE_TABLE_CELL = 'table-cell';
+
+export const countCells = (row: TableRowElement, stop?: number) => {
+  return row.children
+    .map(child => {
+      if (!Element.isElement(child) || child.type !== TYPE_TABLE_CELL) {
+        return 0;
+      }
+      return child.data.colspan ? parseInt(child.data.colspan) : 1;
+    })
+    .slice(0, stop)
+    .reduce((a, b) => a + b);
+};
+
+export const defaultTableBlock = (height: number, width: number) => {
+  return jsx(
+    'element',
+    { type: TYPE_TABLE },
+    [...Array(height)].map(() => defaultTableRowBlock(width)),
```

Comment:
Kan forenkles litt:
```suggestion
    Array(height).fill(defaultTableRowBlock(width)),
```
Samme noen andre steder

Reply:
Jeg forsøkte dette først, men da blir hele listen fylt med samme referanse av blokken som fører til bugs i slate. Må på et vis få `defaultTableRowBlock(width)` til å bli kjørt for hvert eneste element i lista.